recorder_root_path: '/home/yyildirim/workspace/data/demonstrations/sfm/'
scenario: 'small_env_changing_s_g/headless'
